packages/daisyui/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@rjsf/daisyui",
+  "version": "6.0.0-alpha.1",
+  "description": "Daisy UI components for react-jsonschema-form",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "scripts": {
```

Can you add these to improve ESM support, matching the changes made in `5.24.7`
```suggestion
  "types": "src/index.ts",
  "exports": {
    ".": {
      "require": "./dist/index.js",
      "import": "./lib/index.js",
      "types": "./lib/index.d.ts"
    },
    "./lib": {
      "require": "./dist/index.js",
      "import": "./lib/index.js",
      "types": "./lib/index.d.ts"
    },
    "./lib/*.js": {
      "require": "./dist/*.js",
      "import": "./lib/*.js",
      "types": "./lib/*.d.ts"
    },
    "./dist": {
      "require": "./dist/index.js",
      "import": "./lib/index.js",
      "types": "./lib/index.d.ts"
    },
    "./dist/*.js": {
      "require": "./dist/*.js",
      "import": "./lib/*.js",
      "types": "./lib/*.d.ts"
    }
  },
  "scripts": {
```